simtools/model/telescope_model.py

```diff
@@ -502,47 +515,47 @@ def exportSingleMirrorListFile(self, mirrorNumber, setFocalLengthToZero):
             return None
 
         fileName = names.simtelSingleMirrorListFileName(
-            self.version,
+            self.modelVersion,
             self.telescopeName,
             mirrorNumber,
             self.label
         )
-        if '_singleMirrorListFilePath' not in self.__dict__:
+        if not hasattr(self, '_singleMirrorListFilePath'):
             self._singleMirrorListFilePaths = dict()
         self._singleMirrorListFilePaths[mirrorNumber] = self._configFileDirectory.joinpath(fileName)
 
-        __, __, diameter, flen, shape = self.mirrors.getSingleMirrorParameters(mirrorNumber)
-
-        with open(self._singleMirrorListFilePaths[mirrorNumber], 'w') as file:
-            file.write('# Column 1: X pos. [cm] (North/Down)\n')
-            file.write('# Column 2: Y pos. [cm] (West/Right from camera)\n')
-            file.write('# Column 3: flat-to-flat diameter [cm]\n')
-            file.write(
-                '# Column 4: focal length [cm], typically zero = adapting in sim_telarray.\n'
-            )
-            file.write(
-                '# Column 5: shape type: 0=circular, 1=hex. with flat side parallel to y, '
-                '2=square, 3=other hex. (default: 0)\n'
-            )
-            file.write(
-                '# Column 6: Z pos (height above dish backplane) [cm], typ. omitted (or zero)'
-                ' to adapt to dish shape settings.\n'
-            )
-            file.write('#\n')
-            file.write('0. 0. {} {} {} 0.\n'.format(
-                diameter,
-                flen if not setFocalLengthToZero else 0,
-                shape
-            ))
+        # Using SimtelConfigWriter
+        self._loadSimtelConfigWriter()
+        self.simtelConfigWriter.writeSingleMirrorListFile(
+            mirrorNumber,
+            self.mirrors,
+            self._singleMirrorListFilePaths[mirrorNumber],
+            setFocalLengthToZero
+        )
     # END of exportSingleMirrorListFile
```

Comment:
Is this pythonic? Have to smile while seeing this comment....

Reply:
This what? I don't get it. Which comment?

Reply:
I meant this one:
```
# END of exportSingleMirrorListFile 
```
Really not important, I though that the indentation was introduced so that this is not needed...